core/templates/dev/head/services/extension-tag-assembler.service.ts

```diff
@@ -33,7 +33,7 @@ export class ExtensionTagAssemblerService {
               private camelCaseToHyphens: CamelCaseToHyphensPipe) {}
   // TODO(#7176): Replace 'any' with the exact type.
   formatCustomizationArgAttrs(
-      element: any, customizationArgSpecs: Object): any {
+      element: JQuery, customizationArgSpecs: any): any {
```

Comment:
Why is this changed to any? Can you add an explanation for the same?

Reply:
This change is mistaken, I'll change for `Object` again! I had change it during the tests but I forgot to revert it.